### PR TITLE
stc: adjust default gas_limit value

### PR DESCRIPTION
**Pull request by dydysy**
## What does this implement/fix? Explain your changes.
调整 stc 默认 gas_limit

## Does this close any currently open issues?  
If it fixes a bug or resolves a feature request, be sure to link to that issue.
https://github.com/OneKeyHQ/TaskHub/issues/1904#issuecomment-866587824

## Pull request type
 
_Put an `x` in the boxes that apply_
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## Where has this been tested?
…

## Any other comments?
…

### CODE REVIEW

**Review by huazhouwang — APPROVED** [review]
127845 感觉挺 magic code 的。。。

**Comment by dydysy** [comment]
本来先想等等讨论结果的。  看你 approve 了，那就 request review 算了。

**Comment by taimanhui** [comment]
这个数字的确是很奇怪，定义成常量吧 =.=

**Review by huazhouwang — DISMISSED** [review]

File: electrum_gui/common/provider/chains/stc/provider.py [hunk 1]
```diff
@@ -33,6 +33,8 @@ def sign_msg_hash(self, data: bytes):
 
 
 class STCProvider(ProviderInterface):
+    STC_DEFAULT_GAS_Limit = 127845
```
Comment (taimanhui) [inline electrum_gui/common/provider/chains/stc/provider.py:36]: 放外面，没必要放类里...最后几个字母忘了大写 😂

Comment (dydysy) [reply electrum_gui/common/provider/chains/stc/provider.py:36]: 我嫩是没发现。  😂

**Review by taimanhui — APPROVED** [review]

**Comment by shuaiLiWang** [comment]
> 127845 感觉挺 magic code 的。。。

这个数字有故事吗~

**Review by shuaiLiWang — APPROVED** [review]

**Comment by dydysy** [comment]
> > 127845 感觉挺 magic code 的。。。
> 
> 这个数字有故事吗~

stc 主网转账，激活地址情况下的最小 gasLimit 。 产品定下的，参照issue。

**Pull request merged**